### DISCUSSION TITLE
Script ändert nichts

### DISCUSSION THREAD

**Discussion by thelakemann** [svenerbe/ecoflow_dynamic_power_adjustment#13] [Q&A]
Hallo,
ich weiss nicht woran es liegt. Das script ist bei mir aktiv, macht allerdings nichts.
Es gibt auch keine Fehlermeldung.
Ich habe alles so gemacht wie beschrieben und es wird ausgeführt. Aber leider passiert nichts auf der EcoFlow Seite.
Darf in der EcoFlow App irgendetwas nicht aktiv sein?

**Comment by svenerbe** [comment]
Glaskugel lesen:-) im ha log sollte ja mindestens ein Eintrag stehen, ob der Service gestartet wurde. Etwas mehr Input solltest du schon schicken. Mit tut Nix ist kann man leider nicht wirklich etwas anfangen. Grüße

**Reply by thelakemann** [reply]
Ja, ich weiß.
Ultra wenig Informationen, sorry.
Ich wollte eigentlich nur wissen ob irgendetwas in der EcoFlow stören könnte damit das Script nicht umgesetzt werden kann.

**Reply by thelakemann** [reply]
Im ha log steht das die Automatisierung erfolgreich ausgeführt werden konnte.

**Reply by ghino18** [reply]
Hi, for see if It worked i passed single value where Is the sensor totalpower variable and see on ecoflow app if max limit change and i check that world. But i have not understad how. If i pass negative Total Power It set 800 maybe i did not ununderstud the value or sigh that totalpower mean

**Reply by thelakemann** [reply]
I'm sorry, I don't understand what you mean

**Comment by thelakemann** [comment]
Im HA log habe ich den Eintrag "Running Script: false" gefunden.

**Comment by thelakemann** [comment]
Hallo, ich habe weiterhin das Phänomen das es keinerlei Veränderung beim Ausführen des Skriptes gibt.
Nun habe ich ein wenig experimentiert.
Wenn ich das Script wie folgt ausführe ändert der Powerstream die Einspeisung auf 0:
service: pyscript.set_ef_powerstream_custom_load_power
data:
  SerialNumber: tatsächliche SN
  TotalPower: "{{ states('sensor.mein richtiger Sensor') | round }}"
  Automation: false

Auch wenn ich TotalPower: 0 mache verändert er es auf 0 im Powerstream.
Alles andere geht nicht. ändere ich die Zahl händisch auf etwas anderes funktioniert es schlichtweg nicht.
Automation: true verändert ebenso nichts

habt ihr noch eine Idee ??

**Comment by thelakemann** [comment]
jeden Tag etwas neues :) ... nun habe ich heraus gefunden das es ein PS gar nicht mag hinter einer Firewall zu sein (z.B. Ubiquiti).
Denn leider, in dem Fall werden wohl einfach die Übertragungen geblockt !
Also ist mein PS nun separiert !
Nun habe ich aber das Problem das jedesmal beim aufrufen des Script der Wert zum letzten Aufruf addiert wird ! Das heisst nach kurzer Zeit bin ich bei 600 Watt Anschlag und regelt sich nicht runter.
Hat jemand eine Idee? 
Firewall würde ich nun ausschliessen ...

--

Every day something new :) ... now I have found out that a PS does not like being behind a firewall (e.g., Ubiquiti).
Unfortunately, in this case, the transmissions are simply blocked!
So my PS is now separated!
However, I now have the problem that each time the script is called, the value is added to the previous call! This means that after a short time I hit a peak of 600 watts and it doesn't regulate down.
Does anyone have an idea?
I would now rule out the firewall ...

**Reply by svenerbe** [reply]
works a designed - your sensor must be have the value of your grid consumption. first time your grid have 100 w - the service will send 100 w. next time your grid have 0 w. service will send 0. means script if just working with your real consumption. if you send all the time i need 100w more than you will end in 6 times on 600W on PS :-) maybe sometime a short look in the script will help. have a nice day.
